tests/framework/extensions/provisioning/creates.go

```diff
@@ -37,10 +40,13 @@ import (
 	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
 const (
-	namespace = "fleet-default"
+	active     = "active"
+	internalIP = "rke2.io/internal-ip"
+	namespace  = "fleet-default"
 
 	rke2k3sAirgapCustomCluster           = "rke2k3sairgapcustomcluster"
 	rke2k3sNodeCorralName                = "rke2k3sregisterNode"
```

Comment:
nit on style: Shouldn't it be rke2K3sAirgapcustomCluster and rke2K3sNodeCorralName because of camel case?

Reply:
I think that's the TOU test case they do, I didn't touch that in this PR though since it wasn't relevant.

Reply:
@susesgartner probably should, but kind of outside the scope of this PR.